app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -48,6 +62,7 @@ def ensure_network_manager
 
       if network_manager
         populate_network_manager_connectivity(auth_url)
+        network_manager.save
```

Comment:
Can you use `.save!` so this will raise an exception and be logged if saving the network manager fails?